### PR TITLE
Release version 1.23

### PR DESCRIPTION
**Pull request by pali**

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -17,3 +17,6 @@ Daniël van Eeden <daniel.vaneeden@booking.com> <git@myname.nl>
 Pali <pali@cpan.org> <pali@users.noreply.github.com>
 Bernt M. Johnsen <bernt.johnsen@oracle.com> <berntm@gmail.com>
 H.Merijn Brand - Tux <h.m.brand@xs4all.nl>
+Matthew Horsfall (alh) <WolfSage@gmail.com> <wolfsage@gmail.com>
+Jan Holčapek <holcapek@gmail.com> <jan.holcapek@gooddata.com>
+E. Choroba <choroba@matfyz.cz> <choroba@users.noreply.github.com>
```
Comment (choroba) [inline .mailmap:22]: The gooddata address won't work anymore. Similarly, I don't see much value in the noreply.github address.

Comment (pali) [reply .mailmap:22]: That is why I updated `.mailmap` file. First specified address on the line replaces all others address on the line when git is showing email addresses in the git log. So gooddata and noreply.github addresses are replaced by the working one.

Comment (pali) [reply .mailmap:22]: Without this change autogenerating list of contributors contains either more entries for one contributor (if commits contain more addresses, e.g. the working one and the noreply.github) or entry with non-working address.

Comment (pali) [reply .mailmap:22]: Ok. Anything else or is release ready?

**Review by choroba — APPROVED** [review]

**Pull request merged**